_data/guides.yaml

```diff
@@ -49,6 +49,12 @@ categories:
       - title: Using Credential Stores With Encrypted Expressions with WildFly
         url: /guides/security-credential-store-enc-exp
         description: How to set up encrypted expressions and use it to replace clear-text sensitive information.
+  - category: Messaging
+    cat-id: messaging
+    guides:
+      - title: Configuring Messaging in High Availability in Wildfly
```

Comment:
I think these sound more natural:

"Configuring High Availability Messaging in WildFly"
"Configuring Messaging with High Availability in WildFly"

In both cases:

s/Wildfly/WildFly